src/Development/Shake/Command.hs

```diff
@@ -207,8 +207,9 @@ fsatraceFiles file = do
     xs <- parseFSAT <$> readFileUTF8 file
     let reader (FSATRead x) = Just x; reader _ = Nothing
         writer (FSATWrite x) = Just x; writer (FSATMove x y) = Just x; writer _ = Nothing
-    frs <- liftIO $ filterM doesFileExist $ nubOrd $ map normalise $ mapMaybe reader xs
-    fws <- liftIO $ filterM doesFileExist $ nubOrd $ map normalise $ mapMaybe writer xs
+        existing f = liftIO . filterM doesFileExist . nubOrd . mapMaybe f
```

Comment:
No, that's entirely fine, assuming that fsatrace generates normalised paths is very reasonable here, since we know the paths originate with fsatrace. I just don't want to assume normalisation later on, as users can call trackWrite themselves (I'm honestly not sure if we do make that assumption or not!)